docs/en/auto_concurrency_limiter.md

```diff
@@ -0,0 +1,153 @@
+# Adaptive current limit
```

Comment:
There're some specific translation that auto-translation cannot catch.

we should replace 'current limit' with Adaptive Load Shedding